### PR TITLE
ERT-904: Local config work cont 2

### PR DESCRIPTION
**Pull request by arielalmendral**
Continuation local config work. Some new functions and documentation.

### CODE REVIEW

File: devel/python/python/ert/enkf/local_obsdata.py [hunk 1]
```diff
@@ -52,6 +52,12 @@ def addNode(self, node):
         already_exists_node_for_key = LocalObsdata.cNamespace().add_node(self, node)
         return already_exists_node_for_key
 
+    def delNode(self, key):
```
Comment (jepebe) [inline devel/python/python/ert/enkf/local_obsdata.py]: Suggestion:
Python has an operator **__delitem__(self, key)** similiar to **__getitem__**. 

Then you can delete items like this:
**del local_obsdata[key]**  

https://docs.python.org/2/reference/datamodel.html#emulating-container-types

Comment (jepebe) [inline devel/python/python/ert/enkf/local_obsdata.py]: local_obsdata.py should also use registerObjectType

**Comment by arielalmendral** [comment]
Thanks. Please let me know if your comments have been addressed.

**Comment by jepebe** [comment]
Good. If you are going to continue working on the local_* classes I would really like to see more tests.

**Comment by arielalmendral** [comment]
Thanks. There will be more tests coming.

**Pull request merged**